### PR TITLE
croc: new package at 8.3.1

### PR DESCRIPTION
**Pull request by mmatongo**
## Description of package
Easily and securely send things from one computer to another.

## Installed manifest of package

```
/var/db/kiss/installed/croc/version
/var/db/kiss/installed/croc/sources
/var/db/kiss/installed/croc/manifest
/var/db/kiss/installed/croc/depends
/var/db/kiss/installed/croc/checksums
/var/db/kiss/installed/croc/build
/var/db/kiss/installed/croc/
/var/db/kiss/installed/
/var/db/kiss/
/var/db/
/var/
/usr/share/licenses/croc/LICENSE
/usr/share/licenses/croc/
/usr/share/licenses/
/usr/share/
/usr/bin/croc
/usr/bin/
/usr/
```

## New package

- [Y] Latest upstream version.
- [Y] I agree to maintain this package.
- [Y] I agree to notify @dylanaraps if I can no longer maintain this package.

### CODE REVIEW

File: community/croc/build [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+export CGO_CPPFLAGS="${CPPFLAGS}"
+export CGO_CFLAGS="${CFLAGS}"
+export CGO_CXXFLAGS="${CXXFLAGS}"
+export CGO_LDFLAGS="${LDFLAGS}"
+export GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=vendor -modcacherw"
+
+go build
+
+install -Dm755 croc "$1/usr/bin/croc"
+install -Dm644 LICENSE "$1/usr/share/licenses/croc/LICENSE"
```
Comment (dylanaraps) [inline community/croc/build:12]: No need to install the license.

File: community/croc/build [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh -e
+
+export CGO_CPPFLAGS="${CPPFLAGS}"
+export CGO_CFLAGS="${CFLAGS}"
+export CGO_CXXFLAGS="${CXXFLAGS}"
+export CGO_LDFLAGS="${LDFLAGS}"
```
Comment (dylanaraps) [inline community/croc/build:6]: Change `${VAR}` to `$VAR`.

File: community/croc/build [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export GOFLAGS="-buildmode=pie -trimpath -ldflags=-linkmode=external -mod=vendor -modcacherw"
```
Comment (dylanaraps) [inline community/croc/build:7]: Change to `export GOFLAGS="$GOFLAGS ..."`

File: community/croc/build (hunk 1 above)
Comment (mmatongo) [reply community/croc/build:12]: fixed in new pull.

**Comment by mmatongo** [comment]
new pull made

**Pull request closed without merging**